.eslintrc.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
 
 		// rule has no support for our @/ aliases;
 		// the designated replacement, plugin:import, was a pain so we ditched it
+		'n/no-missing-import': 'off',
```

Comment:
If this was renamed, can we remove the old one?

Reply:
Sadly not - it's still there. :shrug:

Reply:
ah. Just tested locally and it seems gone. I can remove this.